src/hooks/use-chat-context.tsx

```diff
@@ -120,7 +120,9 @@ export const ChatProvider: React.FC< ChatProviderProps > = ( { children } ) => {
 			! initialLoad[ selectedSite.id ] &&
 			isCurrent &&
 			! pluginsList[ selectedSite.id ] &&
-			! themesList[ selectedSite.id ]
+			! themesList[ selectedSite.id ] &&
+			selectedSite.running === true &&
+			selectedSite.isAddingSite === false
```

Comment:
`isAddingSite` can also be undefined, so the condition should be `! selectedSite.isAddingSite` instead. Otherwise, themes and plugins are not fetched for new created site.